flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliStringsITCase.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.client.cli;
+
+import org.apache.flink.shaded.guava18.com.google.common.collect.ImmutableSet;
+import org.apache.flink.shaded.guava18.com.google.common.reflect.ClassPath;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/** CliStrings test class. */
+public class CliStringsITCase {
+
+    private final String space = " ";
+    private final String empty = "";
+    final String lineBreaker = "\n";
+
+    /**
+     * Test {@link org.apache.flink.table.client.cli.CliStrings#MESSAGE_HELP} contains all commands'
+     * help descriptions in package {@link org.apache.flink.table.operations.command}.
+     *
+     * @throws IOException exception.
+     */
+    @Test
+    public void testCompletenessOfCliClientCommandsInHelp() throws IOException {
```

Comment:
Personally, I think this test is not necessary and hard to maintain, because class names and packages may change in the future. I think tests like `sql-client-help-command.out` is enough.